ethminer/CMakeLists.txt

```diff
@@ -16,7 +16,10 @@ endif()
 hunter_add_package(CLI11)
 find_package(CLI11 CONFIG REQUIRED)
 
-target_link_libraries(ethminer PRIVATE ethcore poolprotocols devcore ethminer-buildinfo CLI11::CLI11 Boost::system)
+hunter_add_package(Boost COMPONENTS system thread)
```

Comment:
These two lines should not be needed as this is already done in the main CMakeLists.txt file.

Reply:
good catch! 😄  
* removed unnecessary lines
* verifie it compiles

--> ec5b097